pennylane/devices/default_qubit.py

```diff
@@ -86,11 +86,57 @@ class DefaultQubit(QubitDevice):
             returns analytical results.
     """
 
-    name = "Default qubit PennyLane plugin"
-    short_name = "default.qubit"
-    pennylane_requires = __version__
-    version = __version__
-    author = "Xanadu Inc."
+    @property
+    def name(self):
+        return "Default qubit PennyLane plugin"
+
+    @property
+    def short_name(self):
+        """Returns the string used to load the device.
+
+        .. warning::
+
+            Directly using this property is not recommended and should be avoided where possible.
+            The API of DefaultQubit will be updated soon to follow a new interface described
```

Comment:
```suggestion
            The API of ``DefaultQubit`` will be updated soon to follow a new interface described
```
Though this is in many lines below :sweat_smile: